### PR TITLE
Update downsample with notebooks

### PR DESCRIPTION
**Pull request by sanderson**
- [x] Rebased/mergeable

### CODE REVIEW

File: content/influxdb/v2.1/notebooks/downsample.md [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+
+4.  In the Flux script cell:
+    
+    1.  Use `__PREVIOUS_RESULT__` to load the output of the previous notebook
+        cell into the Flux script.
+    3.  Use [`aggregateWindow()`](/{{< latest "flux" >}}/stdlib/universe/aggregateWindow/)
+        to window data into one hour intervals and return the average of each hour.
+        Specify the following parameters:
+
+        - **every**: Window interval _(should be less than or equal to the selected query time range)_.
```
Comment (kelseiv) [inline content/influxdb/v2.1/notebooks/downsample.md:52]:
```suggestion
        - **every**: Window interval _(should be within the selected query time range)_.
```

File: content/influxdb/v2.1/notebooks/downsample.md [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+    
+    Click {{% icon "notebook-add-cell" %}} after your **Flux Script** cell to 
+    add a new cell and select **{{% caps %}}Output to Bucket{{% /caps %}}**.
+    Select a bucket from the **{{% icon "bucket" %}} Choose a bucket**
+    drop-down list. 
+
+    {{% /oss-only %}}
+    
+5.  _(Optional)_ Click {{% icon "notebook-add-cell" %}} and select **Note** to
+    add a cell containing notes about what this notebook does. For example, the
```
Comment (kelseiv) [inline content/influxdb/v2.1/notebooks/downsample.md:91]:
```suggestion
    add a note to describe your notebook, for example, 
```

File: content/influxdb/v2.1/notebooks/downsample.md [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+    cell might say, "This notebook downsamples Coinbase bitcoin prices into hourly averages."
```
Comment (kelseiv) [inline content/influxdb/v2.1/notebooks/downsample.md:92]:
```suggestion
 "Downsample Coinbase bitcoin prices into hourly averages."
```

File: content/influxdb/v2.1/notebooks/downsample.md [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+6.  {{% oss-only %}}
+
+    Click **Preview** in the upper left to verify that your notebook runs and previews the output.
```
Comment (kelseiv) [inline content/influxdb/v2.1/notebooks/downsample.md:95]:
```suggestion
    Click **Preview** in the upper left to verify that your notebook runs and displays the output.
```

File: content/influxdb/v2.1/notebooks/downsample.md [hunk 5]
```diff
@@ -0,0 +1,113 @@
...
+    
+    {{% /oss-only %}}
+6.  Click **Run** to run the notebook and write the downsampled data to your bucket.
+
+## Continuously run a notebook
```
Comment (kelseiv) [inline content/influxdb/v2.1/notebooks/downsample.md:100]: Thoughts on _Continuously write data using a notebook_...possibly more explicit, but maybe not necessary?

File: content/influxdb/v2.1/notebooks/downsample.md [hunk 6]
```diff
@@ -0,0 +1,113 @@
...
+To continuously run your notebook, export the notebook as a task:
+
+1.  Click {{% icon "notebook-add-cell" %}} to add a new cell and then select
```
Comment (kelseiv) [inline content/influxdb/v2.1/notebooks/downsample.md:103]:
```suggestion
1.  Click {{% icon "notebook-add-cell" %}} to add a new cell, and then select
```

File: content/influxdb/v2.1/notebooks/downsample.md [hunk 7]
```diff
@@ -0,0 +1,113 @@
...
+4.  In the Flux script cell:
+    
+    1.  Use `__PREVIOUS_RESULT__` to load the output of the previous notebook
+        cell into the Flux script.
+    3.  Use [`aggregateWindow()`](/{{< latest "flux" >}}/stdlib/universe/aggregateWindow/)
+        to window data into one hour intervals and return the average of each hour.
+        Specify the following parameters:
+
+        - **every**: Window interval _(should be less than or equal to the selected query time range)_.
+        - **fn**: [Aggregate]({{< latest "flux" >}}/function-types/#selectors)
```
Comment (kelseiv) [inline content/influxdb/v2.1/notebooks/downsample.md:53]:
```suggestion
        - **fn**: [Aggregate]({{< latest "flux" >}}/function-types/#aggregates)
```

File: content/influxdb/v2.1/notebooks/downsample.md [hunk 8]
```diff
@@ -0,0 +1,113 @@
...
+          or [selector function]({{< latest "flux" >}}/function-types/#selectors)
```
Comment (kelseiv) [inline content/influxdb/v2.1/notebooks/downsample.md:54]:
```suggestion
          or [selector]({{< latest "flux" >}}/function-types/#selectors) function
```

Comment (kelseiv) [reply content/influxdb/v2.1/notebooks/downsample.md:54]: curious, is something off with latest flux tag? routing to this on localhost:
/influxdb/cloud/notebooks/downsample/flux/v0.x/function-types/#selectors

same with curious, is something off with latest flux tag? routing to this on localhost:
/influxdb/cloud/notebooks/downsample/flux/v0.x/function-types/#aggregates

**Review by kelseiv — COMMENTED** [review]
couple more link comments/questions

File: content/influxdb/v2.1/notebooks/downsample.md [hunk 9]
```diff
@@ -0,0 +1,113 @@
...
+    identifier: notebooks-downsample
+    parent: Notebooks
+---
+
+Downsample data to summarize data at specified time intervals and reduce the
+overall disk usage as data collects over time.
+Learn how to create a notebook that downsamples data.
+
+The following example creates a notebook that queries **Coinbase bitcoin price
+sample data**, downsamples the data into hourly summaries, and then writes the
```
[the remark this answers is not in the document]
Comment (sanderson) [reply content/influxdb/v2.1/notebooks/downsample.md:20]: This makes it sound like it's downsampling on a schedule (like the notebook is scheduled).

File: content/influxdb/v2.1/notebooks/downsample.md (hunk 1 above)
Comment (sanderson) [reply content/influxdb/v2.1/notebooks/downsample.md:49]: "within" feels loaded here. A time range can be defined by timestamps or relative durations. What about this:

```suggestion
        - **every**: Window interval _(should be less than or equal to the duration of the queried time range)_.
```

File: content/influxdb/v2.1/notebooks/downsample.md (hunk 8 above)
Comment (sanderson) [reply content/influxdb/v2.1/notebooks/downsample.md]: Yeah, I missed the first `/` before the shortcode. I fixed them.

File: content/influxdb/v2.1/notebooks/downsample.md (hunk 5 above)
Comment (sanderson) [reply content/influxdb/v2.1/notebooks/downsample.md:97]: Notebooks don't necessarily write data. This particular example does. I don't know that it's necessary to say that.

File: content/influxdb/v2.1/notebooks/downsample.md (hunk 1 above)
Comment (kelseiv) [reply content/influxdb/v2.1/notebooks/downsample.md:49]: "loaded" meaning unclear? :-)
 yes, that works....my thought was the relation wasn't super correlative: less than 1-2pm

**Review by kelseiv — APPROVED** [review]
<s>to be sure, did you see my link comment about aggregates and selectors? it looked like those links were still routing to the /influxdb directory...is that expected when running on localhost?
</s>
see you added the forward slash

File: content/influxdb/v2.1/notebooks/downsample.md (hunk 9 above)
[the remark this answers is not in the document]
Comment (sanderson) [reply content/influxdb/v2.1/notebooks/downsample.md:20]: Ah, ok, so I need to explain a little better. I'll update the description and be more explicit in the instructions.

**Pull request merged**